.github/CONTRIBUTING.md

```diff
@@ -62,6 +64,20 @@ If you want to create a kata for some topic, start by checking the roadmap to se
 If somebody is already working on this topic, you can try to find them (using the repository issues) and coordinate with them.
 If the topic you want is not claimed, or is not on the list, go ahead and let us know you'll be working on it by creating an issue.
 
+### Testing the Katas
+
+You can use Jupyter notebook front-end of the kata you're working on to validate the kata (i.e., to check that all tasks have correct reference solutions for them, and that all tests used in the notebook actually exist in the project.
+
+To validate the kata, use the [`scripts/validate-notebooks.ps1`](../scripts/validate-notebooks.ps1) script. 
+For example, to validate BasicGates kata run the following command from the root directory of the QuantumKatas project:
+
+```bash
+   $ ./scripts/validate-notebooks.ps1 ./BasicGates/BasicGates.ipynb
+```
+
+To use this script, you need to be able to [run Q# Jupyter notebooks locally](https://docs.microsoft.com/quantum/install-guide/jupyter) 
+and to [have PowerShell installed](https://docs.microsoft.com/powershell/scripting/install/installing-windows-powershell).
```

Comment:
The linked URL here is specifically for installing Windows PowerShell, rather than PowerShell Core. Very similar to the .NET Framework vs .NET Core distinction, Windows PowerShell is Windows-only while PowerShell Core runs cross-platform and will be the next version of PowerShell even on Windows (PowerShell 7 is the next version of PowerShell Core, just as .NET 5 is the next version of .NET Core).

Cross-platform instructions for installing PowerShell Core or PowerShell 7 can be found at https://github.com/PowerShell/PowerShell#get-powershell. For users with the .NET Core SDK 2.1 or later already installed, `dotnet tool install --global PowerShell` should work as well.